pkg/ottl/contexts/ottlspanevent/span_events.go

```diff
@@ -47,6 +48,9 @@ func (tCtx TransformContext) MarshalLogObject(encoder zapcore.ObjectEncoder) err
 	err = errors.Join(err, encoder.AddObject("span", logging.Span(tCtx.span)))
 	err = errors.Join(err, encoder.AddObject("spanevent", logging.SpanEvent(tCtx.spanEvent)))
 	err = errors.Join(err, encoder.AddObject("cache", logging.Map(tCtx.cache)))
+	if tCtx.eventIndex != nil {
+		encoder.AddInt64("eventindex", *tCtx.eventIndex)
```

Comment:
@TylerHelmuth, do we have any defined pattern for  the logging field names? It seems the `log` context is using underscores.

Reply:
I used the protobuf names I believe.

Reply:
```suggestion
		encoder.AddInt64("event_index", *tCtx.eventIndex)
```

I see, in that case a nit suggestion would be to use `event_index` instead, like it's doing for the path name and error messages.